### ISSUE TITLE
Missing UI

### ISSUE THREAD

**Issue by lgaetz** [phwhite/texttospeech#1]
@phwhite

I have installed ver. 2.0.0.0 of this module on this brand new PIAF machine set up yesterday:

```
   ┌────────────────────────SYSTEM INFORMATION───────────────────────────┐
   │  Asterisk   = ONLINE  | Dahdi     = ONLINE  | MySQL     = ONLINE    │
   │  SSH        = ONLINE  | Apache    = ONLINE  | Iptables  = ONLINE    │
   │  Fail2ban   = ONLINE  | Internet  = ONLINE  | Ip6Tables = ONLINE    │
   │  Disk Free  = ADEQUATE| Mem Free  = ADEQUATE| NTPD      = ONLINE    │
   │  SendMail   = ONLINE  | Samba     = OFFLINE | Webmin    = ONLINE    │
   │  Ethernet0  = ONLINE  | Ethernet1 = N/A     | Wlan0     = N/A       │
   │                                                                     │
   │  PIAF Installed Version   = 2.0.6.2 under *HARDWARE*                │
   │  FreePBX Version          = 2.9.0.12                                │
   │  Running Asterisk Version = 1.8.8.0                                 │
   │  Asterisk Source Version  = 1.8.8.0                                 │
   │  Dahdi Source Version     = 2.6.0+2.6.0                             │
   │  Libpri Source Version    = 1.4.12                                  │
   │  IP Address               = 10.41.2.154 on eth0                     │
   │  Operating System         = CentOS release 6.2 (Final)              │
   │  Kernel Version           = 2.6.32-220.7.1.el6.i686 - 32 Bit        │
   │                                                                     │
   └─────────────────────────────────────────────────────────────────────┘
```

When clicking on the Text to Speech link under Setup, Applications, Text to Speech, I get this screen:
https://lh4.googleusercontent.com/-w90k-baxSfE/T5fZsNy-UVI/AAAAAAAAA7g/PGEpajL3ljo/s574/ttsmod.jpg

Nothing else.  All the FreePBX menus are gone as well.

I scanned thru the notes from your PIAF forum posting and didn't see anything about dependencies.  I realize this is not much to go on, but I don't have a lot of experience diagnosing these type of issues, what else can I provide to track down this problem?

**Comment by lgaetz** [comment]
Found this in the httpd error log:

```
[Wed Apr 25 08:04:05 2012] [error] [client 10.50.0.180] PHP Fatal error:  Call to undefined function form_open() in /var/www/html/admin/modules/texttospeech/page.texttospeech.php on line 103, referer: http://10.41.2.154/admin/config.php?type=setup&display=index
which: no espeak in (/sbin:/usr/sbin:/bin:/usr/bin)
which: no espeak in (/var/lib/asterisk/bin)
which: no google-tts in (/sbin:/usr/sbin:/bin:/usr/bin)
which: no ms-tts in (/sbin:/usr/sbin:/bin:/usr/bin)
which: no swift in (/sbin:/usr/sbin:/bin:/usr/bin)
which: no swift in (/var/lib/asterisk/bin)
which: no espeak in (/sbin:/usr/sbin:/bin:/usr/bin)
which: no espeak in (/var/lib/asterisk/bin)
which: no google-tts in (/sbin:/usr/sbin:/bin:/usr/bin)
which: no ms-tts in (/sbin:/usr/sbin:/bin:/usr/bin)
which: no swift in (/sbin:/usr/sbin:/bin:/usr/bin)
which: no swift in (/var/lib/asterisk/bin)
which: no espeak in (/sbin:/usr/sbin:/bin:/usr/bin)
which: no espeak in (/var/lib/asterisk/bin)
which: no google-tts in (/sbin:/usr/sbin:/bin:/usr/bin)
which: no ms-tts in (/sbin:/usr/sbin:/bin:/usr/bin)
which: no swift in (/sbin:/usr/sbin:/bin:/usr/bin)
which: no swift in (/var/lib/asterisk/bin)
```

**Comment by phwhite** [comment]
Thank you for your time in searching the logs, as that pointed me in the right direction for getting this issue fixed.  As I posted on the PIAF forum, I had failed to point out in my first post that I had not done any testing on FreePBX 2.9.  As the log hinted, the form_helper routines are not available in FreePBX 2.9.   I was able to work around this, along with adding more detailed debugging to my module, and have updated it to version 2.0.0.2  You can find all the details at the PIAF form, and can find the new version in the github downloads.

Again, I really appreciate the time and effort you put in to trying out the module, and giving me as much feedback as you could!  Let me know if you find the time to try the new version out, and if you hit any other issues.
